### PR TITLE
Create 2024-06-10-crisis-toolkit.md

### PR DESCRIPTION
**Pull request by jmealing**

### CODE REVIEW

**Comment by donohuech** [comment]
The description on the main page should be: Meeting user needs on Canada.ca during a crisis.

File: en/_posts/2024-06-10-crisis-toolkit.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+<h2>Creating a structure so people can find content</h2>
+<ul>
+<li>Just because information is available doesn’t mean it will be found or read.</li>
+<li>Using familiar terms in links and making navigation pages easy to scan helps users find what they’re looking for more quickly and accurately.</li>
+<li>In a crisis, people are more likely to visit from their phones. Design the navigation for mobile, rather than favouring desktop.</li>
+</ul>
+
+<p>For more information about creating structure to help people find content, read:</p>
+<ul>
+<li>Flexible navigation page <a href="https://docs.google.com/document/d/1AXU_Igv6ZAkVOifpMgwld0zQNXSzib0aSxpSIdsI0lw/edit#heading=h.y8x0q33ditvt">(draft)</a></li>
```
Comment (donohuech) [inline en/_posts/2024-06-10-crisis-toolkit.md:55]: Please remove: (draft)

Please add the link to the Flexible navigation blog post, it is: https://blog.canada.ca/2024/05/13/flexible-navigation-page-design-for-emergencies.html

File: fr/_posts/2024-06-10-boite-outils-durgence.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+altLangPage: /2024/06/10/crisis-toolkit.html
+css:
+-  ../../../css/blog.css
+date: 2024-06-10
+dateModified: 2024-06-10
+description: "Répondre aux besoins des utilisateurs et utilisatrices de Canada.ca en temps de crise."
+title: "Boîte à outils de conception pour les situations d’urgence"
```
Comment (donohuech) [inline fr/_posts/2024-06-10-boite-outils-durgence.md:8]: On the preview the H1 is displaying as: "Boîte à outils de conception pour les situations d’urgence: Blogue de Canada.ca"

Can you please remove ": Blogue de Canada.ca"

File: fr/_posts/2024-06-10-boite-outils-durgence.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<h2>Créer une structure permettant aux gens de trouver du contenu</h2>
+<ul>
+<li>Un contenu peut être disponible sans pour autant être facile à trouver ou à lire.</li>
+<li>L’utilisation de termes courants et la simplification des pages de navigation afin de faciliter le balayage visuel permettent aux utilisateurs et utilisatrices de trouver rapidement les renseignements recherchés.</li>
+<li>En temps de crise, les gens sont plus susceptibles de visiter les sites Web sur leur téléphone. Il faut donc concevoir le contenu en vue d’une navigation sur mobile plutôt que de privilégier la version pour ordinateur.</li>
+</ul>
+
+<p>Pour plus de renseignements sur la création d’une structure permettant aux gens de trouver du contenu, consultez les pages suivantes&nbsp;:</p>
+<ul>
+<li>Des pages de navigation flexibles pour les situations d’urgence <a href="https://docs.google.com/document/d/1YF6qGdgjzFrKVdnXBD_wRp_29tUyF4TNOE3w50opTKE/edit#heading=h.gjdgxs">(brouillon)</a></li>
```
Comment (donohuech) [inline fr/_posts/2024-06-10-boite-outils-durgence.md:55]: Please remove: (brouillon)

Please instead link to the blog post, the link is: https://blogue.canada.ca/2024/05/13/conception-de-page-de-navigation-flexible-pour-les-urgences

**Review by donohuech — CHANGES_REQUESTED** [review]

File: fr/_posts/2024-06-10-boite-outils-durgence.md (hunk 3 above)
Comment (jmealing) [reply fr/_posts/2024-06-10-boite-outils-durgence.md]: Fixed

File: en/_posts/2024-06-05-unified-design-system.md
Comment (delisma) [inline en/_posts/2024-06-05-unified-design-system.md:1]: This modification should be in a different PR

File: fr/_posts/2024-06-05-systeme-design-unifie.md
Comment (delisma) [inline fr/_posts/2024-06-05-systeme-design-unifie.md:1]: This modification should be in a different PR

**Review by delisma — CHANGES_REQUESTED** [review]

File: fr/_posts/2024-06-10-boite-outils-durgence.md (hunk 2 above)
Comment (jmealing) [reply fr/_posts/2024-06-10-boite-outils-durgence.md:8]: It was a problem with the CSS, fixed now

File: fr/_posts/2024-06-05-systeme-design-unifie.md
Comment (delisma) [inline fr/_posts/2024-06-05-systeme-design-unifie.md:1]: This file should be in a different PR

**Review by delisma — CHANGES_REQUESTED** [review]

File: fr/_posts/2024-06-10-boite-outils-durgence.md (hunk 2 above)
Comment (donohuech) [reply fr/_posts/2024-06-10-boite-outils-durgence.md:8]: I still see it it in the H1: https://deploy-preview-163--blogue-sct.netlify.app/2024/06/10/boite-outils-durgence 
Blog_H1

File: fr/_posts/2024-06-05-systeme-design-unifie.md
Comment (jmealing) [reply fr/_posts/2024-06-05-systeme-design-unifie.md:1]: I just undid my changes, how else do I get rid of it?

**Review by donohuech — APPROVED** [review]

File: fr/_posts/2024-06-05-systeme-design-unifie.md
Comment (delisma) [reply fr/_posts/2024-06-05-systeme-design-unifie.md:1]: 🤔

Comment (delisma) [reply fr/_posts/2024-06-05-systeme-design-unifie.md:1]: Line four is supposed to be `-  ../../../css/blogue.css` instead of `-  ../../../css/blog.css` when I compare with the upstream.

Comment (jmealing) [reply fr/_posts/2024-06-05-systeme-design-unifie.md:1]: Thanks, I think I had too many tabs open and forgot to save. Fixed

**Review by delisma — APPROVED** [review]
🚀 LGTM
